### PR TITLE
User on-boarding completed (dashboard/setup)

### PR DESCRIPTION
**Pull request by nikkaroraa**

### CODE REVIEW

File: src/routes/dashboard.js [hunk 1]
```diff
@@ -1,11 +1,80 @@
...
+  */
+  const { currentUser } = req;
+  const db = await getDb();
+  const collection = productionConstants.USERS_COLLECTION;
+  // update user details
+  const response = await updateUser(db, collection, currentUser, req.body);
+  res.json(response);
+});
+
+// route.post('/organisation/create', (req, res) => {
```
Comment (ArfatSalman) [inline src/routes/dashboard.js]: Why comments?

**Review by ArfatSalman — CHANGES_REQUESTED** [review]

File: src/routes/dashboard.js (hunk 1 above)
Comment (iamoperand) [reply src/routes/dashboard.js]: I was making a layout for the next feature.

Comment (ArfatSalman) [reply src/routes/dashboard.js]: Then put it in a different branch..

Comment (ArfatSalman) [reply src/routes/dashboard.js]: or in git stash ..

Comment (iamoperand) [reply src/routes/dashboard.js]: Done!

**Pull request merged**